### PR TITLE
[KOGITO-6794] - Create configmap for every supporting service route

### PR DESCRIPTION
**Pull request by vaibhavjainwiz**
Many thanks for submitting your Pull Request :heart:! 

Jira : https://issues.redhat.com/browse/KOGITO-6794, https://issues.redhat.com/browse/KOGITO-1940

Please make sure your PR meets the following requirements:

- [ ] You have read the [contributors' guide](https://github.com/kiegroup/kogito-operator/blob/main/README.md#contributing-to-the-kogito-operator)
- [ ] Pull Request title is properly formatted: `[KOGITO-XYZ] Subject`
- [ ] Pull Request contains a link to the JIRA issue
- [ ] Pull Request contains a description of the issue
- [ ] Pull Request does not include fixes for issues other than the main ticket
- [ ] Your feature/bug fix has a unit test that verifies it
- [ ] You've ran `make before-pr` and everything is working accordingly
- [ ] You've tested the new feature/bug fix in an actual OpenShift cluster
- [ ] You've added a [RELEASE_NOTES.md](https://github.com/kiegroup/kogito-operator/blob/main/RELEASE_NOTES.md) entry regarding this change

<details>
<summary>
How to retest this PR or trigger a specific build:
</summary>

* <b>Run operator BDD testing</b>
  Please add comment: <b>/jenkins test</b>

* <b>Run RHPAM operator BDD testing</b>
  Please add comment: <b>/jenkins RHPAM test</b>
</details>

### CODE REVIEW

**Review by sutaakar — APPROVED** [review]
Looks good to me, seems to work as expected

**Review by spolti — APPROVED** [review]

**Comment by spolti** [comment]
```
TASK [operator_info : Fail if 'cluster_type' is not defined] *******************
fatal: [localhost]: FAILED! => changed=false 
  msg: Variable 'cluster_type' is mandatory.
```

@vaibhavjainwiz seems to be the sabe issue you have reported, right?
https://github.com/redhat-openshift-ecosystem/operator-test-playbooks/issues/305

File: core/connector/url_handler.go [hunk 1]
```diff
@@ -97,152 +100,116 @@ func (u *urlHandler) InjectDataIndexURLIntoSupportingService(namespace string, s
...
-				}
-			}
-		}
 	}
-	u.Log.Debug("Service Endpoint is nil")
 	return nil
 }
 
-// InjectDataIndexURLIntoDeployment will inject Supporting service route URL in to kogito runtime deployment env var
+// InjectDataIndexEndpointOnDeployment will inject Supporting service route URL in to kogito runtime deployment env var
```
Comment (ricardozanini) [inline core/connector/url_handler.go:186]:
```suggestion
// injectSupportingServiceURLIntoDeployment will inject Supporting service route URL in to kogito runtime deployment env var
```

**Review by ricardozanini — APPROVED** [review]

File: core/connector/url_handler.go (hunk 1 above)
Comment (vaibhavjainwiz) [reply core/connector/url_handler.go:186]: done

**Comment by vaibhavjainwiz** [comment]
> ```
> TASK [operator_info : Fail if 'cluster_type' is not defined] *******************
> fatal: [localhost]: FAILED! => changed=false 
>   msg: Variable 'cluster_type' is mandatory.
> ```
> 
> @vaibhavjainwiz seems to be the sabe issue you have reported, right? [redhat-openshift-ecosystem/operator-test-playbooks#305](https://github.com/redhat-openshift-ecosystem/operator-test-playbooks/issues/305)

yes @spolti Its a same issue.

**Review by radtriste — APPROVED** [review]

**Pull request merged**